ui/src/ctkrangeslider.cpp

```diff
@@ -769,7 +769,7 @@ void ctkRangeSlider::mouseReleaseEvent(QMouseEvent* mouseEvent)
   this->QSlider::mouseReleaseEvent(mouseEvent);
 
   setSliderDown(false);
-  d->m_SelectedHandles = 0;
+  d->m_SelectedHandles = ctkRangeSliderPrivate::NoHandle;
```

Comment:
Where does this come from? I can't find it.

Reply:
it's defined in this same file, line 42

Reply:
Alright, was looking in the .h file